CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * `PickupModal`: Show count of items awaiting pickup based on `totalRecords` API response field. Refs UICHKOUT-846.
 * UI tests replacement with RTL/Jest for src/components/CheckoutNoteModal/CheckoutNoteModal.js. Refs UICHKOUT-825.
 * UI tests replacement with RTL/Jest for src/components/MultipieceModal/MultipieceModal.js. Refs UICHKOUT-828.
+* UI tests replacement with RTL/Jest for src/components/PatronBlock/PatronBlockModal.js. Refs UICHKOUT-832.
```

Comment:
Could you please update `CHANGELOG.md `? example below:
## in progress

* UI tests replacement with RTL/Jest for src/components/PatronBlock/PatronBlockModal.js. Refs UICHKOUT-832.


## [9.0.0](https://github.com/folio-org/ui-checkout/tree/v9.0.0) (2023-02-22)
[Full Changelog](https://github.com/folio-org/ui-checkout/compare/v8.2.0...v9.0.0)